src/acceleration/AitkenAcceleration.cpp

```diff
@@ -47,14 +47,9 @@ void AitkenAcceleration::initialize(DataMap &cplData)
   Eigen::VectorXd toAppend    = Eigen::VectorXd::Constant(entries, initializer);
   utils::append(_residuals, toAppend);
 
-  // Append column for old values if not done by coupling scheme yet
-  for (DataMap::value_type &pair : cplData) {
-    int cols = pair.second->oldValues.cols();
-    if (cols < 1) {
-      PRECICE_ASSERT(pair.second->values().size() > 0, pair.first);
-      utils::append(pair.second->oldValues,
-                    (Eigen::VectorXd) Eigen::VectorXd::Zero(pair.second->values().size()));
-    }
+  for (DataMap::value_type &pair : cplData) { // @todo: seems to be unnecessary, since this is again just a loop over CouplingScheme::getAccelerationData
+    PRECICE_ASSERT(pair.second->values().size() > 0, pair.first);
+    pair.second->storeIteration();
```

Comment:
Coupling data and acceleration data can be different, see https://precice.org/configuration-acceleration.html
If it is really not needed, let's remove it. Will only get more complicated if we don't solve it now.

Reply:
Solved in 4046df9b6a92019d634b952f4ff8e1bc8523940a.